be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/validators/ElectionValidator.scala

```diff
@@ -57,7 +72,11 @@ object ElectionValidator extends MessageDataContentValidator with EventValidator
         }*/
         // FIXME: check the actual votes
         // FIXME: for the VoteElection list, we need to check question ids but what do they mean? No info in documentation
-        else {
+        else if (!validateAttendee(sender, channel)){
+          Right(validationError(s"invalid PoP token"))
```

Comment:
Can the error message be more expressive something like : "Attendee $sender has an invalide PoP token -- is not present in LAO"

Reply:
I believe that since this error message will only concern people outside the LAO, we do not need to provide much more information

Reply:
However, I formulated the message better